cfgov/cfgov/settings/base.py

```diff
@@ -649,7 +649,7 @@
     "'self'",
     "*.consumerfinance.gov",
     "*.google-analytics.com",
-    "*.tiles.mapbox.com",
```

Comment:
Are you certain that it's ok to remove this across the site?

Reply:
Pretty sure, but as I'd prefer to get this in quickly before running a full audit, I added it back.